xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-ui/src/main/resources/XWiki/DeletedAttachments.xml

```diff
@@ -168,8 +183,8 @@ $response.setContentType('application/json')
 #end
 #set($selectSql = "select datt.id, $order from DeletedAttachment as datt where 1=1 " + $sql + " order by $order $dir")
 #set($countSql = "select count(*) from DeletedAttachment as datt where 1=1 " + $sql)
-#set($items = $services.query.hql($selectSql).setLimit($limit).setOffset($offset).execute())
-#set($totalItems = $services.query.hql($countSql).execute().get(0))
+#set($items = $services.query.hql($selectSql).bindValues($params).setLimit($limit).setOffset($offset).execute())
```

Comment:
I guess you checked and it works when ``$params`` is empty ``[]`` (i.e. when there's no date filter specified) right?